kctf-operator/pkg/controller/challenge/challenge_controller.go

```diff
@@ -108,37 +108,36 @@ func (r *ReconcileChallenge) Reconcile(request reconcile.Request) (reconcile.Res
 		r.client.Delete(ctx, challenge)
```

Comment:
I think you need `return reconcile.Result{}, nil` here for my suggested change to work? So that in case when we have invalid name we stop executing the fuction

Reply:
That's right, forgot about that. Fixed!